docs/source/using/addons.adoc

```diff
@@ -71,6 +71,11 @@ running as well. This is similar to running `eval $(minishift docker-env)` on yo
 then executing any `docker` command. See also
 link:../command-ref/minishift_docker-env{outfilesuffix}[`minishift docker-env`].
 
+echo::
+If the add-on command starts with _echo_, the arguments following the _echo_ command are printed
+to the console. This can be used to provide additional feedback during add-on execution or to print
+a user hint at the end of the execution.
```

Comment:
I think we should just keep `This can be used to provide additional feedback during add-on execution` and remove `or to print a user hint at the end of the execution` as this is not required explicitly IMO.